mmv1/templates/terraform/examples/scc_event_threat_detection_custom_module.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_scc_event_threat_detection_custom_module" "<%= ctx[:primary_resource_id] %>" {
+  organization = "<%= ctx[:test_env_vars]['org_id'] %>"
+  display_name = "<%= ctx[:vars]['display_name'] %>"
+  enablement_state = "ENABLED"
+  type = "<%= ctx[:vars]['type'] %>"
```

Comment:
I think this needs to be added the the ruby example block

Reply:
Done